taxi/urls.py

```diff
@@ -22,6 +22,8 @@
     path(
         "drivers/<int:pk>/", DriverDetailView.as_view(), name="driver-detail"
     ),
+    path("login/", auth_views.LoginView.as_view(), name="login"),
+    path("logout/", auth_views.LogoutView.as_view(), name="logout"),
```

Comment:
When using the default Django authentication system, you don't need to explicitly declare every view in your urls.py. However, if you want to customize the URLs or templates used for authentication views, or if you want to have fine-grained control over how these views are presented in your application, you would need to declare them.